config/crd/bases/exp.cluster.x-k8s.io_machinepools.yaml

```diff
@@ -30,6 +30,10 @@ spec:
       jsonPath: .status.phase
       name: Phase
       type: string
+    - description: Kubernetes version associated with this MachinePool
+      jsonPath: .status.version
```

Comment:
@Evalle you need to run make generate to update this

Reply:
Done